Two test regressions with ufo2ft 3.7.1

```
$ git checkout https://github.com/googlefonts/glyphsLib.git
$ cd glyphsLib
$ uv venv
$ . .venv/bin/activate
(glyphsLib) $ uv pip install -e .[ufo_normalization,defcon,colr]
(glyphsLib) $ uv pip install -r requirements-dev.in
(glyphsLib) $ uv pip list | grep ufo2ft
ufo2ft            3.7.1
(glyphsLib) $ pytest -v
[…]
========================================================================================= FAILURES =========================================================================================
__________________________________________________________________________________ test_ignorable_anchors __________________________________________________________________________________

datadir = local('/home/ben/src/forks/glyphsLib/tests/data')

    def test_ignorable_anchors(datadir):
        ufos = load_to_ufos(datadir.join("IgnorableAnchors.glyphs"))

        for ufo in ufos:
            writer = ContextualMarkFeatureWriter()
            feaFile = ast.FeatureFile()
            assert str(feaFile) == ""
            assert writer.write(ufo, feaFile)

            assert len(feaFile.markClasses) == 1
>           assert "MC_top" in feaFile.markClasses
E           AssertionError: assert 'MC_top' in {'mark_top': <fontTools.feaLib.ast.MarkClass object at 0x7f8823ab2490>}
E            +  where {'mark_top': <fontTools.feaLib.ast.MarkClass object at 0x7f8823ab2490>} = <fontTools.feaLib.ast.FeatureFile object at 0x7f881f14fef0>.markClasses

tests/feature_writers_test.py:74: AssertionError
_________________________________________________________________________________ test_contextual_anchors __________________________________________________________________________________

datadir = local('/home/ben/src/forks/glyphsLib/tests/data')

    def test_contextual_anchors(datadir):
        ufos = load_to_ufos(datadir.join("ContextualAnchors.glyphs"))

        for ufo in ufos:
            writer = ContextualMarkFeatureWriter()
            feaFile = ast.FeatureFile()
            assert str(feaFile) == ""
            assert writer.write(ufo, feaFile)

            assert len(feaFile.markClasses) == 2
>           assert "MC_bottom" in feaFile.markClasses
E           AssertionError: assert 'MC_bottom' in {'mark_bottom': <fontTools.feaLib.ast.MarkClass object at 0x7f8823a1a2c0>, 'mark_top': <fontTools.feaLib.ast.MarkClass object at 0x7f88206da650>}
E            +  where {'mark_bottom': <fontTools.feaLib.ast.MarkClass object at 0x7f8823a1a2c0>, 'mark_top': <fontTools.feaLib.ast.MarkClass object at 0x7f88206da650>} = <fontTools.feaLib.ast.FeatureFile object at 0x7f881f66ff50>.markClasses

tests/feature_writers_test.py:17: AssertionError
================================================================================= short test summary info ==================================================================================
FAILED tests/feature_writers_test.py::test_ignorable_anchors - AssertionError: assert 'MC_top' in {'mark_top': <fontTools.feaLib.ast.MarkClass object at 0x7f8823ab2490>}
FAILED tests/feature_writers_test.py::test_contextual_anchors - AssertionError: assert 'MC_bottom' in {'mark_bottom': <fontTools.feaLib.ast.MarkClass object at 0x7f8823a1a2c0>, 'mark_top': <fontTools.feaLib.ast.MarkClass object at 0x7f88206da650>}
================================================================== 2 failed, 1159 passed, 64 skipped, 4 xfailed in 5.32s ===================================================================
(glyphsLib) $ uv pip install 'ufo2ft==3.7.0'
(glyphsLib) $ pytest -v
[…]
======================================================================= 1161 passed, 64 skipped, 4 xfailed in 4.66s ========================================================================
```

--- a/tests/feature_writers_test.py
+++ b/tests/feature_writers_test.py
@@ -2,6 +2,7 @@ from textwrap import dedent
 from glyphsLib import load_to_ufos
 from glyphsLib.featureWriters.markFeatureWriter import ContextualMarkFeatureWriter
 from ufo2ft.featureWriters import ast
+from ufo2ft.featureWriters.markFeatureWriter import MarkFeatureWriter
 
 
 def test_contextual_anchors(datadir):
@@ -14,7 +15,7 @@ def test_contextual_anchors(datadir):
         assert writer.write(ufo, feaFile)
 
         assert len(feaFile.markClasses) == 2
-        assert "MC_bottom" in feaFile.markClasses
+        assert f"{MarkFeatureWriter.markClassPrefix}_bottom" in feaFile.markClasses
 
         feature = feaFile.statements[-1]
         assert feature.name == "mark"
@@ -38,7 +39,7 @@ def test_contextual_anchors(datadir):
             "    lookupflag UseMarrkFilteringSet [twodotshorizontalbelow];\n"
             "    # reh-ar * behDotess-ar.medi &\n"
             "    pos reh-ar [behDotless-ar.init] behDotess-ar.medi"
-            " @MC_bottom' lookup ContextualMark_0;\n"
+            f" @{MarkFeatureWriter.markClassPrefix}_bottom' lookup ContextualMark_0;\n"
             "} ContextualMarkDispatch_0;\n"
         )
 
@@ -48,7 +49,7 @@ def test_contextual_anchors(datadir):
             "    lookupflag UseMarrkFilteringSet [twodotsverticalbelow];\n"
             "    # reh-ar *\n"
             "    pos reh-ar [behDotless-ar.init behDotless-ar.init.alt]"
-            " @MC_bottom' lookup ContextualMark_1;\n"
+            f" @{MarkFeatureWriter.markClassPrefix}_bottom' lookup ContextualMark_1;\n"
             "} ContextualMarkDispatch_1;\n"
         )
 
@@ -56,7 +57,8 @@ def test_contextual_anchors(datadir):
         assert str(lookup) == (
             "lookup ContextualMarkDispatch_2 {\n"
             "    # reh-ar *\n"
-            "    pos reh-ar [behDotless-ar.init] @MC_bottom' lookup ContextualMark_2;\n"
+            "    pos reh-ar [behDotless-ar.init] "
+            f"@{MarkFeatureWriter.markClassPrefix}_bottom' lookup ContextualMark_2;\n"
             "} ContextualMarkDispatch_2;\n"
         )
 
@@ -71,7 +73,7 @@ def test_ignorable_anchors(datadir):
         assert writer.write(ufo, feaFile)
 
         assert len(feaFile.markClasses) == 1
-        assert "MC_top" in feaFile.markClasses
+        assert f"{MarkFeatureWriter.markClassPrefix}_top" in feaFile.markClasses
 
         feature = feaFile.statements[-2]
         assert feature.name == "mark"
@@ -82,4 +84,6 @@ def test_ignorable_anchors(datadir):
         for statement in lookup.statements:
             assert isinstance(statement, ast.MarkBasePosStatement)
             assert len(statement.marks) == 1
-            assert statement.marks[0][1].name == "MC_top"
+            assert (
+                statement.marks[0][1].name == f"{MarkFeatureWriter.markClassPrefix}_top"
+            )

